get-started/graph-database/graph-database.adoc

```diff
@@ -20,93 +20,89 @@ You should have a basic understanding of what a database is.
-
-image::{img}graph_description.svg[]
-
-_The building blocks of the Property Graph_
-
-There is one core consistent rule in a graph database: *"No broken links"*.  Since a relationship always has a start and end node, you can’t delete a node without also deleting its associated relationships.  You can also always assume that an existing relationship will never point to a non-existing endpoint.
-
+Due to the efficient way relationships are stored, two nodes can share any number or type of relationships without sacrificing performance.
+Although they are stored in a specific direction, relationships can always be navigated efficiently in either direction.
 
+.Building blocks of the property graph model
+image:{img}graph_description.svg[]
 
-=== What is Neo4j?
 
-Neo4j is an open-source NoSQL native graph database which provides an ACID-compliant transactional backend for your applications.  With development starting in 2003, it has been publicly available since 2007.  The source code, written in Java and Scala, is available on https://github.com/neo4j/neo4j[GitHub], with a thriving community on the https://neo4j.com/slack[Neo4j Slack] and https://stackoverflow.com/tags/neo4j/[StackOverflow].
+== What is Neo4j?
 
-Neo4j is https://neo4j.com/customers/[used today] by thousands of companies and organizations in almost all industries, including financial services, government, energy, technology, retail and manufacturing.  Hundreds of developers and architects in those industries are https://neo4j.com/graphacademy/neo4j-certification/[Neo4j Certified Professionals].
+Neo4j is an open-source, NoSQL, native graph database that provides an ACID-compliant transactional backend for your applications.
+Initial development began in 2003, but it has been publicly available since 2007.
+The source code, written in Java and Scala, is available for free on https://github.com/neo4j/neo4j[GitHub^] or as a user-friendly https://neo4j.com/download/[desktop application download^].
 
-[role=side-nav]
-* https://www.youtube.com/watch?v=nCnR6wRo8x4[Video: Neo4j Graph Platform]
-* link:/developer/guide-data-modeling/[Guide: Intro to Graph Modeling]
-* https://www.youtube.com/watch?v=5Tl8WcaqZoc&list=PL9Hl4pk2FsvWM9GWaguRhlCQ-pa-ERd4U[Video Series: Intro to Graph Databsaes]
-* link:/graphacademy/online-training/getting-started-graph-databases-using-neo4j/[Training: Online Intro Course]
-* http://graphdatabases.com/[Book: O'Reilly Graph Databases (free)]
+Neo4j is referred to as a native graph database because it efficiently implements the property graph model down to the storage level.
+This means that the data is stored exactly as you whiteboard it, and the database uses pointers to navigate and traverse the graph.
+In contrast to graph processing or in-memory libraries, Neo4j also provides full database characteristics, including ACID transaction compliance, cluster support, and runtime failover - making it suitable to use graph data in production scenarios.
```

Comment:
edit: "...making it suitable to use **graphs for data** ~~graph~~ in production scenarios.